packages/KTable/KTable.vue

```diff
@@ -184,6 +217,19 @@ export default {
 
 table.k-table {
   border-collapse: collapse;
+
+  &.side-border {
+    border-collapse: separate;
+    border-spacing: 0 5px;
```

Comment:
this should be 4px.
![image](https://user-images.githubusercontent.com/5941111/86270245-ca730e00-bb7f-11ea-9d38-9ef697ff0a35.png)